tests/pages/FicheJeune.page.test.tsx

```diff
@@ -563,6 +600,93 @@ describe('Fiche Jeune', () => {
         })
       })
 
+      describe('affiche l’agenda du jeune', () => {
+        it('avec un message si le jeune n’a rien sur la période', async () => {
+          // Given
+          await act(async () => {
+            await renderWithContexts(
+              <FicheJeune
+                jeune={jeune}
+                rdvs={[]}
+                actionsInitiales={{
+                  actions,
+                  page: 1,
+                  metadonnees: { nombreTotal: 14, nombrePages: 2 },
+                }}
+                pageTitle={''}
+                metadonneesFavoris={metadonneesFavoris}
+              />,
+              {
+                customConseiller: { structure: StructureConseiller.MILO },
+                customDependances: dependances,
+              }
+            )
+          })
+
+          // Then
+          expect(
+            screen.getByRole('tab', {
+              name: /Agenda/,
+            })
+          ).toBeInTheDocument()
+          expect(
+            screen.getByText(
+              'Il n’y a pas encore de rendez-vous ni d’action prévus sur cette période.'
+            )
+          ).toBeInTheDocument()
+        })
+
+        it('avec un des rendez-vous et des actions si le jeune en a sur la période', async () => {
```

Comment:
nitpick :-)
```suggestion
        it('avec des rendez-vous et des actions si le jeune en a sur la période', async () => {
```
ou c'est vraiment voulu, en mode "l'un des rendez-vous"? mais en fixture c'est 1 rdv, 1 action